toolkit/tools/pkg/imagecustomizerlib/runscripts.go

```diff
@@ -29,6 +32,13 @@ func runUserScripts(baseConfigPath string, scripts []imagecustomizerapi.Script,
 
 	logger.Log.Infof("Running %s scripts", listName)
 
+	_, span := otel.GetTracerProvider().Tracer(OtelTracerName).Start(ctx, "run_user_scripts")
+	span.SetAttributes(
+		attribute.Int("scripts_count", len(scripts)),
+		attribute.String("list_name", listName),
```

Comment:
I would change the name of the event to: `run_postcustomization_scripts` and `run_finalizeimage_scripts` instead of logging `listName`.

Reply:
updated, thanks